### PR TITLE
JPW-419 Create a TimeOrderMixin for models with a time_order column.

### PR DESCRIPTION
**Pull request by ray-ray**
Move fixtures into conftest

### CODE REVIEW

File: jhhalchemy/tests/integration/conftest.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import flask
+import flask_sqlalchemy
+import pytest
+import sqlalchemy
+import sqlalchemy_utils
+
+#
+# Set your own MYSQL connection string here
+#
+MYSQL_CONNECTION_URI = 'mysql://root:root@0.0.0.0/test_db'
```
Comment (imjohsep) [inline jhhalchemy/tests/integration/conftest.py]: Parameterize credentials for testings.

File: jhhalchemy/tests/integration/conftest.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    if not sqlalchemy_utils.database_exists(engine.url):
+        sqlalchemy_utils.create_database(engine.url)
+
+    #
+    # Initialize flask and flask_sqlalchemy
+    #
+    app = flask.Flask(__name__)
+    app.config['SQLALCHEMY_DATABASE_URI'] = MYSQL_CONNECTION_URI
+
+    import jhhalchemy.model
```
Comment (imjohsep) [inline jhhalchemy/tests/integration/conftest.py]: Can import be at the top of the file?

**Review by imjohsep — APPROVED** [review]

File: jhhalchemy/tests/integration/conftest.py (hunk 1 above)
Comment (ray-ray) [reply jhhalchemy/tests/integration/conftest.py]: Good idea. Allowing override with an ENV var.

File: jhhalchemy/tests/integration/conftest.py (hunk 2 above)
Comment (ray-ray) [reply jhhalchemy/tests/integration/conftest.py]: Previously, it couldn't, but I just tried to move it now that it's in `conftest.py`, and it works!

**Review by imjohsep — APPROVED** [review]

**Pull request merged**